ui/pages/onboarding-flow/pin-extension/pin-extension.js

```diff
@@ -75,7 +75,7 @@ export default function OnboardingPinExtension() {
       await dispatch(setCompletedOnboarding());
 
       if (externalServicesOnboardingToggleState) {
-        if (!isProfileSyncingEnabled || participateInMetaMetrics) {
+        if (isProfileSyncingEnabled || participateInMetaMetrics) {
```

Comment:
lol.

Not sure why this was inverted...

Reply:
I think the idea was only sign in if you are going from false to true... but realistically we want to check both correctly.